### ISSUE TITLE
Syntax error after compiling treeBreaker

### ISSUE THREAD

**Issue by maesaar** [ansariazim/treeBreaker#7]
I compiled treeBreaker but it gives me error (using with test data).

```
./treeBreaker -x 1000 -y 1000 -z 100 testTree.newick phenoTestFile.txt output
Syntax error.
        Usage: treeBreaker [OPTIONS] inputfile_tree inputfile_phenotype outputfile
        
        Options:
        -x NUM      Sets the number of iterations after burn-in (default is 500000)
        -y NUM      Sets the number of burn-in iterations (default is 500000)
        -z NUM      Sets the number of iterations between samples (default is 1000)
        -S NUM      Sets the seed for the random number generator to NUM
        -v          Verbose mode
```

**Comment by tamelgarejo** [comment]
I am having the same issue... any idea how to fix it?... thanks

**Comment by ansariazim** [comment]
Hi there,

Which version are you running. I would say use version 1.0  which you can dowload from here:

https://github.com/ansariazim/treeBreaker/tags

Then just follow the instruction in README document included with the v1.0.

Let me know if you still have issues.

**Comment by tamelgarejo** [comment]
Thank you a lot!... now it is working very nice..
